hello-nextflow/projectC/main.nf

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env nextflow
+
+/*
+ * Pipeline parameters
+ */
+
+// Primary input (file of input files, one per line)
+params.reads_bam = "${projectDir}/data/sample_bams.txt"
+
+// Accessory files
+params.reference        = "${projectDir}/data/ref/ref.fasta"
+params.reference_index  = "${projectDir}/data/ref/ref.fasta.fai"
+params.reference_dict   = "${projectDir}/data/ref/ref.dict"
+params.intervals        = "${projectDir}/data/ref/intervals.bed"
+
+// Base name for final output file
+params.cohort_name = "family_trio"
+
+/*
+ * Generate BAM index file
+ */
+process SAMTOOLS_INDEX {
+
+    container 'community.wave.seqera.io/library/samtools:1.20--b5dfbd93de237464'
```

Comment:
We usually group process directives together. It makes it easier to read.

Reply:
I find it more readable to have them separate, tbh... Going to dismiss for now (also because I'm not keen to update all version of the script at this time) but let's discuss at some point to align on best practices.